docs/en/02_Developer_Guides/14_Files/03_File_Security.md

```diff
@@ -0,0 +1,375 @@
+summary: Manage access permission to assets
+
+# File Security
+
+## Security overview
+
+File security is an important concept, and is as essential as managing any other piece of data that exists 
+in your system. As pages and dataobjects can be either versioned, or restricted to view by authenticated
+members, it is necessary at times to apply similar logic to any files which are attached to these objects
+in the same way.
+
+Out of the box SilverStripe Framework comes with an asset storage mechanism with two stores, a public
+store and a protected one. Most operations which act on assets work independently of this mechanism,
+without having to consider whether any specific file is protected or public, but can normally be
+instructed to favour private or protected stores in some cases.
+
+For instance, in order to write an asset to a protected location you can use the following additional
+config option:
+
+
+    :::php
+    $store = singleton('AssetStore');
+    $store->setFromString('My protected content', 'Documents/Mydocument.txt', null, null, array(
+        'visibility' => AssetStore::VISIBILITY_PROTECTED
+    ));
+
+
+## User access control
+
+Access for files is granted on a per-session basis, rather than on a per-member basis, via a
+whitelist action. This means that access to any protected asset must be made prior to the user
```

Comment:
Suggest reword to "via whitelisting accessed assets" (since Michael didn't pick up on this as per his comments)